modules/addons/ratelimit/main.tf

```diff
@@ -9,7 +9,7 @@ provider "helm" {
 resource "random_password" "redis" {
   length = 16
   # Do not use the ':' character here as it will be read as Basic Auth and tokenized as <user>:<password>
-  override_special = "!@#$%&*()-_=+[]{}<>?"
+  special = false
```

Comment:
Are there other characters apart from the `:` that cause trouble?

Reply:
If I recall correct, as I was multi-tasking with the customer - I had password with `{`, so decided to keep it simple, also searched around looks it is a common issue, found a reference here as well: https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/auth.html - decided to keep it simple, as technically 16 symbols should cover the password complexity requirements.